assets/js/components/KeyMetrics/ChipTabGroup/index.js

```diff
@@ -190,12 +237,24 @@ export default function ChipTabGroup( { allMetricItems, savedItemSlugs } ) {
 		select( CORE_UI ).getValue( KEY_METRICS_SELECTION_PANEL_OPENED_KEY )
 	);
 	const isSelectionPanelOpenPrevious = usePrevious( isSelectionPanelOpen );
+	const newlyDetectedMetricsKeys = Object.keys( newlyDetectedMetrics );
 
 	useEffect( () => {
 		// Ensure that current selection group is always active when selection panel re-opens.
 		if ( ! isSelectionPanelOpenPrevious && isSelectionPanelOpen ) {
-			setIsActive( KEY_METRICS_CURRENT_SELECTION_GROUP_SLUG );
-			setActiveGroupIndex( 0 );
+			if ( newlyDetectedMetricsKeys.length && isMobileBreakpoint ) {
```

Comment:
This part is included, since on mobile the selection panel should set new group as active, instead of current selection, opposed to how it work on other breakpoints